src/meili/map_matcher.cc

```diff
@@ -492,21 +492,21 @@ struct path_t {
 namespace valhalla {
 namespace meili {
 
-MapMatcher::MapMatcher(const boost::property_tree::ptree& config,
+MapMatcher::MapMatcher(const Config& config,
```

Comment:
I don't think the API compatibility should be an issue, I've also cc'd some downstream users of the mapmatching library in case they have opinions.